README.md

```diff
@@ -63,7 +92,35 @@ An example of three clusters configured to use with Submariner would look like t
 
 # Installation
 
-## Setup
+Submariner installation can be done manually using helm charts or by using its operator. Both methods are described here.
+
+## Installation using operator
+
+An operator is a tool to package, manage and deploy any Kubernetes application. A submariner operator will soon be available on [OperatorHub.io](https://operatorhub.io).
+
+Current implementation builds submariner operator and deploys submariner in a [Kind setup](https://github.com/submariner-io/submariner/blob/master/scripts/kind-e2e/README.md).
```

Comment:
IMO, the main README should be primarily user-oriented. The operator section should match the Helm section: the latter doesn’t explain how to build the Helm charts, it only explains how to install Submariner using the Helm charts. The operator section should only explain how to install Submariner using the operator.

Reply:
Yes I asked to remove that, but not sure where my comment is gone? :-?

Reply:
Done